libcontainer/specconv/spec_linux.go

```diff
@@ -635,11 +643,17 @@ func parseMountOptions(options []string) (int, []int, string) {
 			}
 		} else if f, exists := propagationFlags[o]; exists && f.flag != 0 {
 			pgflag = append(pgflag, f.flag)
+		} else if f, exists := extensionFlags[o]; exists && f.flag != 0 {
+			if f.clear {
+				extFlags &= ^f.flag
+			} else {
+				extFlags |= f.flag
+			}
 		} else {
 			data = append(data, o)
```

Comment:
> There is no implementation of -o delete and -o copyup for now. 

:wink:

But I think we should have a nicer way of doing this than adding to a flag list that might eventually clash with the kernel. Maybe we should add an `extension` field to `mounts`? Or would that be overkill?